js/bin/stats/single-config/null.js

```diff
@@ -0,0 +1,77 @@
+/******/ (function(modules) { // webpackBootstrap
```

Comment:
Ah yeah that should go in `test/js/bin/stats/single-config/null.js` (that folder is ignored)